doc/converse/manual.rst

```diff
@@ -2014,12 +2028,15 @@ callbacks from within ccd callbacks.
 
 .. code-block:: c++
 
-  void CcdRaiseCondition(int condNum)
+  double CcdRaiseCondition(int condNum)
 
 
 When this function is called, it invokes all the functions whose
 pointers were registered for the ``condNum`` via a *prior* call to
-``CcdCallOnCondition`` or ``CcdCallOnConditionKeep``.
+``CcdCallOnCondition`` or ``CcdCallOnConditionKeep``. The function
+internally calls ``CmiWallTimer`` and returns this value. When using
+``CcdRaiseCondition``, the return value can be used to determine the
+current walltime avoiding an additional call to ``CmiWallTimer``.
```

Comment:
You should mention here that the wall time returned may be stale by the time it is returned, because any functions that were registered for that condition will be called between the timer call and the return. So users shouldn't depend on it being exact or current.

Reply:
Done.